### PR TITLE
Tweak the race descriptions of orcs and goblins

### PR DESCRIPTION
**Pull request by CelticMinstrel**
There was some questionable wording which could be easily misunderstood.

### CODE REVIEW

File: data/core/units.cfg [hunk 1]
```diff
@@ -299,7 +299,7 @@ The eastern provinces of Wesnoth, known as the Clan Homelands, have a geography
         male_name= _ "race^Orc"
         female_name= _ "race+female^Orc"
         plural_name= _ "race^Orcs"
-        description= _"In appearance, orcs are half men and half beasts. They are taller, sturdier and stronger than humans. They are warlike, savage, and cruel by nature. Their blood is darker and thicker than that of normal humans and they have little care for personal hygiene or their personal appearance. Although Orcs are violent even among themselves, they are pack-oriented; an orc never travels long or lives in groups smaller than half a dozen.
+        description= _"In appearance, orcs resemble humans but with some bestial features. They are taller, sturdier and stronger than humans. They are warlike, savage, and cruel by nature. Their blood is darker and thicker than that of humans, and they have little care for personal hygiene or their personal appearance. Although Orcs are violent even among themselves, they are pack-oriented; an orc never travels long or lives in groups smaller than half a dozen.
```
Comment (stevecotton) [inline data/core/units.cfg]: "never travels long" sounds wrong to me, "never travels long distances" or  "never travels alone" both sound better

Comment (CelticMinstrel) [reply data/core/units.cfg]: Oh, good point.

**Comment by doofus-01** [comment]
What do you think about removing the part about litters?  It implies they have rows of nipples, makes them less humanoid.  They can still have high fertility rates without litters.

There is also a spelling mistake:
> The rest, often a full half of more of any litter, will be goblins.

of -> or

**Comment by CelticMinstrel** [comment]
I don't really have a problem with the litters part? I don't think it requires them to be less humanoid, either.

**Comment by doofus-01** [comment]
What does it add, aside from weirdness?

**Pull request closed without merging**